android/app/src/main/java/com/emergetools/hackernews/features/comments/CommentsScreen.kt

```diff
@@ -235,21 +239,23 @@ private fun CommentsScreenLoadingPreview() {
 fun CommentRow(
   modifier: Modifier = Modifier,
   state: CommentState,
+  onToggleHide: (CommentState.Content) -> Unit,
   onLikeTapped: (CommentState.Content) -> Unit
 ) {
-  val startPadding = (state.level * 16).dp
-  Column(
-    modifier = modifier
-      .padding(start = startPadding)
-      .fillMaxWidth()
-      .heightIn(min = 80.dp)
-      .clip(RoundedCornerShape(8.dp))
-      .background(color = MaterialTheme.colorScheme.surface)
-      .padding(8.dp),
-    verticalArrangement = Arrangement.spacedBy(8.dp)
-  ) {
-    when (state) {
-      is CommentState.Content -> {
+  when (state) {
+    is CommentState.Content -> {
+      val startPadding = (state.level * 16).dp
+      Column(
+        modifier = modifier
+          .padding(start = startPadding)
+          .fillMaxWidth()
+          .wrapContentHeight()
+          .clip(RoundedCornerShape(8.dp))
+          .clickable { onToggleHide(state) }
```

Comment:
Thoughts on clickable vs long click? No strong preference, but worried clicking might be too sensitive

Reply:
Based on our convo, we'll keep this as is for now. Seems to be convention (according to the stock reddit app).